src/components/ItemTile/ItemTile.module.scss

```diff
@@ -1,10 +1,46 @@
 @import '../../assets/styles/theme';
 
+body{
+  background-color: pink;
+}
+  
+//   .image {
```

Comment:
nie dodajemy zakomentowanego kody, mozesz ten kod zapisac u siebie np w notatniku lub
klikasz na wcześniejszej kropce (commicie, pod twoim) i klikasz `reset current branch on this commit` i wybierasz KONIECZNIE `mixed` i powinny ci sie pojawić WSZYSTKIE twoje zmiany po lewej stronie w tej zakładce od commitowania

jak to masz to klikasz tam `3 kropki w prawym górnym` i wybierasz `stash` i wpisujesz nazwę i Enter
![image](https://github.com/UEK-i-team/team-homepage/assets/49104061/9641b80b-9af9-466d-9007-b0ae84553735)

twoje stashe są w tej samej zakładce na dole
![image](https://github.com/UEK-i-team/team-homepage/assets/49104061/ef607d77-2d68-476a-8025-e667f8ac8b76)